it/v1/endpoints/AuthISpec.scala

```diff
@@ -44,17 +43,35 @@ class AuthISpec extends IntegrationBaseSpec {
 
     def request(): WSRequest = {
       setupStubs()
-      buildRequest(s"/$nino/$taxYear/sampleEndpoint")
+      buildRequest(s"/$nino/crystallisation")
         .withHttpHeaders((ACCEPT, "application/vnd.hmrc.1.0+json"))
     }
 
-    def desUri: String = s"/income-tax/nino/$nino/taxYear/${DesTaxYear.fromMtd(taxYear)}/someService"
+    def desUri: String = s"/enterprise/obligation-data/nino/$nino/ITSA"
 
     val desResponse: JsValue = Json.parse(
       """
         | {
-        | "responseData" : "someResponse"
-        | }
+        |    "obligations": [
+        |        {
+        |            "identification": {
+        |                "incomeSourceType": "ITSA",
+        |                "referenceNumber": "AB123456A",
+        |                "referenceType": "NINO"
+        |            },
+        |            "obligationDetails": [
+        |                {
+        |                    "status": "F",
+        |                    "inboundCorrespondenceFromDate": "2018-04-06",
+        |                    "inboundCorrespondenceToDate": "2019-04-05",
+        |                    "inboundCorrespondenceDateReceived": "2020-01-25",
+        |                    "inboundCorrespondenceDueDate": "2020-01-31",
+        |                    "periodKey": "ITSA"
+        |                }
+        |            ]
+        |        }
+        |    ]
+        |}
     """.stripMargin)
   }
 
```

Comment:
"Calling the sample endpoint" -> "calling an endpoint"/"calling the crystallisation endpoint"/whatever